download_latest.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+set -o errexit
+
+filename=migration_verifier
+
+RELEASE_URL="https://api.github.com/repos/mongodb-labs/migration-verifier/releases/latest"
+
+OS=$(uname -o | tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m)
+if [ "$ARCH" = "aarch64" ]; then
+    ARCH=arm64
+fi
+
+echo "Looks like you’re running $OS on $ARCH."
+
+MANIFEST=$(wget -qO- "$RELEASE_URL")
```

Comment:
The instructions tell people to use `curl` to fetch this script, but then it uses `wget` internally. I think it should `curl` since we know they already have that installed.

Reply:
But maybe this isn't needed at all.